src/lib/Libnet/net_server.c

```diff
@@ -860,25 +831,26 @@ close_conn(int sd)
  * 	Given an index within the svr_conn array, reset all fields back to
  * 	their defaults and clear any select/poll related flags.
  *
- * @param[in]	cndx: index of the svr_conn entry
+ * @param[in]	idx: index of the svr_conn entry
  *
- * @return	void
  */
 static void
-cleanup_conn(int cndx)
+cleanup_conn(int indx)
 {
-	if (tpp_em_del_fd(poll_context, svr_conn[cndx]->cn_sock) < 0) {
-		snprintf(logbuf, sizeof(logbuf), "tpp_em_del_fd, Remove from poll list failed for sock %d\n",
-					svr_conn[cndx]->cn_sock);
-		log_err(-1, __func__, logbuf);
+	int err;
+	if (tpp_em_del_fd(poll_context, svr_conn[indx]->cn_sock) < 0) {
+		err = errno;
```

Comment:
Can we use `errno` in `log_err` directly instead of using an additional variable `err`?

Reply:
The call to snprintf() could change the value of errno, so it gets saved to err before the call.

Reply:
got it. Thanks!

Reply:
Thanks for explaining Mike.